plugins/modules/helm.py

```diff
@@ -139,6 +139,12 @@
     type: bool
     default: False
     version_added: "1.2.0"
+  history_max:
+    description:
+      - Limit the maximum number of revisions saved per release.
+    type: int
+    default: False
```

Comment:
Please create a [changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment). See [this fragment](https://github.com/ansible/ansible/blob/c2f528b7686c9573aa8493357219e0bbe4416dd3/changelogs/fragments/reboot-change-default-boot-command.yaml) as an example.